### DISCUSSION TITLE
Lint ignore for deprecated fields

### DISCUSSION THREAD

**Discussion by roeest** [graphql-hive/graphql-eslint#1226] [General]
Hi everyone,

Im looking to implement a stricter enforcing of graphql schema writing style.
My goal is to point out style discrepancies using the linter, and allow people to deprecate their old types/fields/operations in favor of style adhering ones. However after playing around with the linter i don't see a way to have the linter ignore rules for deprecated fields. Having a bunch of ignore directives in an already massive schema doesn't seem like the cleanest solution.
How difficult would that be to implement? Is there a solution that won't involve modifying all the individual rules?
If this is currently possible in any way i'd love to hear how i can accomplish it.

**Comment by dimaMachina** [comment]
https://github.com/B2o5T/graphql-eslint/blob/master/docs/rules/no-deprecated.md

Is this the rule you are looking for?

**Reply by roeest** [reply]
This seems to be geared towards ensuring client queries aren't using deprecated fields, im trying to analyze a schema file and getting the lint rules to be ignored when the fields are marked as deprecated.
